src/Integration/LocalServices/SolutionBindingInformationProvider.cs

```diff
@@ -45,6 +49,7 @@ public SolutionBindingInformationProvider(IServiceProvider serviceProvider)
         #region ISolutionBindingInformationProvider
         public bool IsSolutionBound()
         {
+            // TODO: CM2: consider changing this method to "IsLegacySolutionBound"
```

Comment:
CM2?

Reply:
Short-hand for "Connected Mode version 2" - a marker to search to tag things I still need to do.